[Menu] Add new context menu demo (#17839)
mui-org/material-ui: docs/src/pages/components/menus/menus.md

{{"demo": "pages/components/menus/FadeMenu.js"}}

> ## Context Menu

Reviewer: ```suggestion
## Context menu
```

> Here is an example of the of Context Menu.

Reviewer: ```suggestion
Here is an example of a context menu. (Right click to open.)
```

> The `ContextMenu` wraps the area in which you'd like right-click functionality. Just add the tag `<ContextMenu></ContextMenu>` around the element and your custom context menu will be available anywhere within that element.

Reviewer 2: Avoid "just" when possible

```suggestion
The `ContextMenu` wraps the area in which you'd like right-click functionality. add the tag `<ContextMenu></ContextMenu>` around the element and your custom context menu will be available anywhere within that element.
```

Reviewer 2: Also, I'm not sure that the description matches the content of the demo.

Author: I will rewrite the description.
